### PR TITLE
pass our CA cert when configuring Pulp3 Client

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

**Comment by ekohl** [comment]
This reminds me I started https://github.com/Katello/katello/pull/8832 a long time ago and I should revisit that.

**Comment by evgeni** [comment]
bleh, this doesn't work as it should, whyyy

**Comment by evgeni** [comment]
> bleh, this doesn't work as it should, whyyy

because @evgeni restarted foreman after changing the code, but not the dynflow worker, and the code talking to Pulp obviously runs there…

File: app/services/cert/certs.rb [hunk 1]
```diff
@@ -4,8 +4,12 @@ def self.ueber_cert(organization)
       organization.debug_cert
     end
 
+    def self.ca_cert_filename
+      Setting[:ssl_ca_file]
```
Comment (evgeni) [inline app/services/cert/certs.rb:8]: but… is that the right file?
it's `/etc/foreman/proxy_ca.pem`, which is the one we use to talk to proxies.
but here, were talking to Pulpcore via Apache, and Apache will be using the server-ca?

@ehelms halp

Comment (ekohl) [reply app/services/cert/certs.rb:8]: I think so, but I still prefer https://github.com/Katello/katello/pull/8832 which defaults to that file while still allowing to override it if needed. It also works it into the ping controller.

**Comment by evgeni** [comment]
Fixed by https://github.com/Katello/katello/pull/8832, thanks Ewoud!

**Pull request closed without merging**